From 01777a6b56f8951c82f13cc7141392b15598edf5 Mon Sep 17 00:00:00 2001
From: Yinbo Zhu <yinbo.zhu@nxp.com>
Date: Sun, 30 Sep 2018 13:58:57 +0800
Subject: [PATCH 605/767] armv8: pm: change error message pop logic

This patch is to remove the err print when rcpm node is null

Signed-off-by: Yinbo Zhu <yinbo.zhu@nxp.com>
[Xulin: Original patch taken from NXP LSDK-18.12.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/soc/fsl/rcpm.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/drivers/soc/fsl/rcpm.c b/drivers/soc/fsl/rcpm.c
index e0c8a1201c72..30d415ce9dfd 100644
--- a/drivers/soc/fsl/rcpm.c
+++ b/drivers/soc/fsl/rcpm.c
@@ -133,10 +133,8 @@ static int __init layerscape_rcpm_init(void)
 	struct device_node *np;
 
 	np = of_find_matching_node_and_match(NULL, rcpm_matches, &match);
-	if (!np) {
-		pr_err("Can't find the RCPM node.\n");
+	if (!np)
 		return -EINVAL;
-	}
 
 	if (match->data)
 		rcpm = (struct rcpm_config *)match->data;
-- 
2.17.0

